docs/Agent/user-guide/man_page.md

```diff
@@ -68,17 +68,17 @@ Show this message and exit.
 Clear all tools which are registered and can filter by name of the group.
 
 **OPTIONS**  
-`-C`, `--config` PATH  
+`-C`, `--config PATH`  
 Path to the Pbench Agent configuration file.
 This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
 
-`-n`, `--name`, `--names` STR  
-Clear only the <`STR`> tool.
+`-n`, `--name`, `--names <name>`  
```

Comment:
I'm not thrilled with the mix of `<placeholder>` and `PLACEHOLDER` syntax, but let's deal with that later!